lib/config.js

```diff
@@ -83,7 +83,11 @@ function loadConfig(filePath) {
                 filePath = "eslint-config-" + filePath;
             }
 
-            config = require(filePath);
+            var extendedConfig = require(filePath);
```

Comment:
This is not extendedConfig as it has nothing to do with extends. Maybe `packagedConfig`?